ext_localconf.php

```diff
@@ -131,4 +131,9 @@
     $GLOBALS['TYPO3_CONF_VARS']['SYS']['locallangXMLOverride']['de']
             ['EXT:frontend/Resources/Private/Language/locallang_tca.xlf'][] =
         'EXT:examples/Resources/Private/Language/de.custom.xlf';
+
+    // Add custom doktype to the page tree toolbar
```

Comment:
We should move this into ext_tables.php as ext_localconf.php is loaded also in frontend context (which is not necessary for user TSconfig) - like before.